src/Scan.js

```diff
@@ -249,8 +250,7 @@ class Scan extends React.Component {
     if (!isEmpty(errors)) {
       return errors;
     }
-
-    const { item: { barcode } } = data;
+    const barcode = '"' + escapeCqlValue(data.item.barcode) + '"';
```

Comment:
@K-Felk do we still need `"` around in this case? if so could you replace it with template literal `` and `"${escapeCqlValue(data.item.barcode)}"`

Reply:
You do still need literal quotation marks-I tried without and the query continued to trigger the ugly error (also the bug report specifically requests it). I can reformat.

Reply:
Interesting.  Eslint does not like that and is throwing an error, "no template curly in string"

Reply:
Ah ok then let's just keep it the way it is. Eslint is annoying sometimes :)